misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -689,6 +709,53 @@ abstract class PathAssertionsSpec(
             expect(folder).isDirectoryFun()
         }
     }
+
+    describeFun(hasSameBinaryContentAs.name, hasSameTextualContentAs.name) {
+        val hasSameBinaryContentAsFun = hasSameBinaryContentAs.lambda
+        val hasSameTextualContentAsFun = hasSameTextualContentAs.lambda
+
+        context("has same binary content") {
+            it("${hasSameBinaryContentAs.name} - does not throw") withAndWithoutSymlink { maybeLink ->
+                val sourcePath = maybeLink.create(tempFolder.tmpDir.resolve("text1"))
+                val targetPath = maybeLink.create(tempFolder.tmpDir.resolve("text2"))
+                expect {
+                    expect(sourcePath).hasSameBinaryContentAsFun(targetPath)
+                }
+            }
+        }
+
+        context("has same textual content") {
```

Comment:
```suggestion
        context("same text but UTF-8 and UTF-16 encoding") {
```

and please add the use case hasSameBinaryContentAs throws to this context